.travis.yml

```diff
@@ -9,14 +9,18 @@ branches:
 notifications:
   email: false
 node_js:
-  - iojs
+  - 0.12.0
+  - 4
+  - 5
```

Comment:
We probably don't need to run stuff on v5. Just stable versions of Node will do I think.